project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject com.rentpath/rp-util-clj "1.0.13-SNAPSHOT"
+(defproject com.rentpath/rp-util-clj "1.0.14-SNAPSHOT"
```

Comment:
There's no need to manually update this. It will be bumped as part of the automated release process.

Reply:
Ah, cheers, wasn't aware.